src/test/java/com/exasol/adapter/jdbc/JdbcAdapterTest.java

```diff
@@ -104,11 +85,27 @@ void testGetCapabilities() throws AdapterException {
                 createSchemaMetadataInfo());
         final ExaMetadata exaMetadataMock = mock(ExaMetadata.class);
         final GetCapabilitiesResponse response = this.adapter.getCapabilities(exaMetadataMock, request);
-        assertThat(response.getCapabilities().getMainCapabilities(), emptyCollectionOf(MainCapability.class));
+        assertThat(response.getCapabilities().getMainCapabilities(), contains(MainCapability.ORDER_BY_EXPRESSION));
     }
 
     @Test
-    void testDropVirtualSchemaMustSucceedEvenIfDebugAddressIsInvalid() throws AdapterException {
+    void testGetCapabilitiesWithExcludedCapabilitiesList() throws AdapterException {
+        setGenericSqlDialectProperty();
```

Comment:
Rename this method to `setTestSqlDialectProperty` please because we are not using the generic dialect anymore and this is confusing.

Reply:
done